src/main/scala/com/mpc/scalats/core/TypeScriptGenerator.scala

```diff
@@ -13,21 +13,24 @@ object TypeScriptGenerator {
 
   def generateFromClassNames(
                               classNames: List[String],
-                              out: PrintStream,
                               classLoader: ClassLoader = getClass.getClassLoader
                             )
                             (implicit config: Config) = {
     implicit val mirror = runtimeMirror(classLoader)
     val types = classNames map { className =>
       mirror.staticClass(className).toType
     }
-    generate(types, out)
+    generate(types)(config)
   }
 
-  def generate(caseClasses: List[Type], out: PrintStream)(implicit config: Config) = {
+  def generate(caseClasses: List[Type])(implicit config: Config) = {
+    val outputStream = if(config.outputFile.isEmpty)
```

Comment:
I kept a PrintStream, I can't really see a use case not covered by it, aside from sbt based dark magic :D